stable/datadog/values.yaml

```diff
@@ -36,9 +36,11 @@ daemonset:
   ## Allow the DaemonSet to schedule on tainted nodes (requires Kubernetes >= 1.6)
   # tolerations: []
 
-  # Node labels for pod assignment
-  # Ref: https://kubernetes.io/docs/user-guide/node-selection/
-  nodeSelector: {}
+  ## Allow the DaemonSet to schedule on selected nodes
+  # nodeSelector: {}
```

Comment:
Can we keep the reference comment from before? It's helpful for users not familiar with node selectors.